Expecta/Matchers/EXPMatchers+raise.m

```diff
@@ -15,13 +15,13 @@
     return expectedExceptionCaught;
   });
 
-  failureMessageForTo(^NSString *{
+  failureMessageForTo(^NSString *(id actual) {
```

Comment:
`actual` is not used in the block. Should it be marked with `__unused`? Same for next block.

Reply:
Done.